### ISSUE TITLE
Can I group the issues by specific label instead of the first label？

### ISSUE THREAD

**Issue by liangweihua-rock** [github-tools/github-release-notes#227]
I got following error when group by specific label, can I do it by any option?
gren's version is 0.17.0
```
gren release --group-by "Type:Bug" --tags=v0.1.0..v0.2.0 --override

🤖  - Generate release notes:
===================================
✔ Releases found: 2
✔ Tags found: v0.2.0, v0.1.0
✔ Issues found: 63
The option for groupBy is invalid, please check the documentation

```

**Comment by alexcanessa** [comment]
Hello @liangweihua-rock `groupBy` is either `label` or an Object (which means you need the config file) see https://github-tools.github.io/github-release-notes/options.html#group-by-1

**Comment by liangweihua-rock** [comment]
> Hello @liangweihua-rock `groupBy` is either `label` or an Object (which means you need the config file) see https://github-tools.github.io/github-release-notes/options.html#group-by-1

Thanks a lot for your reply, I've tried the config file in the format of  https://github-tools.github.io/github-release-notes/options.html#group-by-1, seems the issues is grouped by Label1 OR label2, i.e I want to group the issue with label1 AND label2 in one group, but seems all the issues with lable1 OR label2 are grouped in. I'll try it further.

**Comment by alexcanessa** [comment]
@liangweihua-rock oh I see what you mean.

We don't have any logic for that atm.
Have you thought of a possible API for it?

**Comment by liangweihua-rock** [comment]
> @liangweihua-rock oh I see what you mean.
> 
> We don't have any logic for that atm.
> Have you thought of a possible API for it?

Yes...I hope there would be an API to group the label1 AND label2, but I am not familar with javascript, I am in learning process.

**Comment by alexcanessa** [comment]
@liangweihua-rock I think we could use Arrays for that.

Something like this would group under the group `Enhancements`, the issues that have `(enhancement AND new feature) OR internal`.

```json
{
    "groupBy": {
        "Enhancements:": [["enhancement", "new feature"], "internal"]
    }
}
```
